crates/vm/levm/src/vm.rs

```diff
@@ -711,22 +711,26 @@ impl VM {
 
         let blob_gas_cost = self.get_blob_gas_price()?;
 
-        // The real cost to deduct is calculated as effective_gas_price * gas_limit + value + blob_gas_cost
-        let up_front_cost = gaslimit_price_product
-            .checked_add(value)
-            .ok_or(VMError::TxValidation(
-                TxValidationError::InsufficientAccountFunds,
-            ))?
-            .checked_add(blob_gas_cost)
-            .ok_or(VMError::TxValidation(
-                TxValidationError::InsufficientAccountFunds,
-            ))?;
+        // The real cost to deduct is calculated as:
+        // effective_gas_price * gas_limit + value + blob_gas_cost
+        let up_front_cost_without_blobgas =
```

Comment:
I am a not a fan of this variable name. But, I didn't want the fact that we are _not_ adding `blob_gas_cost` to be in a comment.

What do you think?